soyjs/exec.go

```diff
@@ -293,7 +292,7 @@ func (s *state) visitPrint(node *ast.PrintNode) {
 	s.indent()
 	s.js(s.bufferName, " += ")
 	for _, dir := range directives {
-		s.js("soy.$$", dir.Name, "(")
+		s.js(PrintDirectives[dir.Name].Name, "(")
```

Comment:
this turns into a syntax error for |id or |noAutoescape, right?  I wonder if we can ignore them instead